### DISCUSSION TITLE
ana sayfa "/" hatası

### DISCUSSION THREAD

**Discussion by Ezagor-dev** [realsuayip/django-sozluk#40] [Q&A]
Merhabalar, 

Ana sayfam normalde çalışırken bir anda 502 hatası almaya başladım. Sadece ana sayfada hata alıyorum. Geri kalan fonksiyonlarım çalışıyor.Tam olarak hatayı sizinle paylaşıyorum. 

2022/02/17 12:37:02 [error] 1273859#1273859: *1 upstream prematurely closed connection while reading response header from upstream, client: 5.27.16.37, server: mobilasyon.com, request: "GET / HTTP/1.1", upstream: "http://unix:/run/gunicorn.sock:/", host: "www.mobilasyon.com", referrer: "http://www.mobilasyon.com/admin/"

Bunu nasıl çözebileceğimi biliyor musunuz acaba ? 

Şimdiden teşekkür ederim.

**Comment by EchoBilisim** [comment]
Hocam merhaba kurulum yapmışsınız bende kurmak istiyorum kurulum hakkında yardımcı olabilirmisiniz acaba ?
